code/modules/cm_aliens/structures/fruit.dm

```diff
@@ -148,7 +148,7 @@
 	mature = FALSE
```

Comment:
Please also set picked true here so the message can be corrected from destroyed to consumed if you eat fruit w/o picking up the fruit.